### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_Keyboard: Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7618]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Regression From

NO

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#Voiceover;#WCAG2.4.7;#Focus Visible;#AILimited;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Open Connect dialog" pane and Sign into Azure.
3. Tab Navigate to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Select "Blob Containers" of the selected subscription.
5. Tab Navigate to "Properties" Tab present on left bottom of the screen and hit enter.
6. Observe that Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.

### Actual Experience

Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.

### Expected Experience

Keyboard focus should be visible properly for Custom Domain, Allow protected append writers text box.

### User Impact:
Keyboard users will face difficulty if Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.

### Attachment:
![A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_Keyboard](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/4857b81f-d92d-450b-8826-ac45cc003f26)

**Comment by MRayermannMSFT** [comment]
An outline will now be visible on the inputs so focus can be visibly tracked:
![7623](https://github.com/microsoft/AzureStorageExplorer/assets/21112697/8c687466-51b5-4895-a11f-e7f3a04b09d5)

**Comment by keerthiduvvuri** [comment]
@MRayermannMSFT Can you please let us know in which version this bug is fixed, so that i can verify and close it.

**Comment by kupatkar99** [comment]
verified the issue in below environment:
Version: 1.33.0-dev (96)
Platform: macOS
Architecture: x64
Build Number: 20240111.7
issue is fixed . hence Closing the bug please find added attachment for reference.
<img width="1440" alt="image (3)" src="https://github.com/microsoft/AzureStorageExplorer/assets/98530069/31e90e47-a478-4d35-a87d-b271b45a5b88">
